include/gatekeeper_net.h

```diff
@@ -388,10 +391,10 @@ struct net_config {
 	gid_t                pw_gid;
 
 	/*
-	 * There is a memory pool per NUMA node to be used for
+	 * There is a memory pool per lcore to be used for
 	 * packet buffers in that node.
 	 */
-	struct rte_mempool   **gatekeeper_pktmbuf_pool;
+	struct rte_mempool   *gatekeeper_pktmbuf_pool[RTE_MAX_LCORE];
```

Comment:
Field `gatekeeper_pktmbuf_pool` is no longer needed. Right now, it's only needed to enable `get_pktmbuf_pool()`, but if we move the creation of the mbuf pools to the blocks, there's no need for `get_pktmbuf_pool()`. Moreover, having field `gatekeeper_pktmbuf_pool` is now a liability because each block should only access its own pool because the pools only support one consumer (i.e. any mismatch can be fatal).